### PR TITLE
Prepare release notes for v1.0.0-rc2

### PR DESCRIPTION
**Pull request by llucax**

### CODE REVIEW

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 ## Summary
 
-<!-- Here goes a general summary of what this release is about -->
+This version ships an experimental version of the **Power Manager**, add preliminary support for n:m relations between inverters and batteries and add user documentation.
```
Comment (daniel-zullo-frequenz) [inline RELEASE_NOTES.md]:
```suggestion
This version ships an experimental version of the **Power Manager**, adds preliminary support for n:m relations between inverters and batteries and includes user documentation.
```

**Review by daniel-zullo-frequenz — COMMENTED** [review]
I have a cosmetic comment and LGTM

File: RELEASE_NOTES.md (hunk 1 above)
Comment (llucax) [reply RELEASE_NOTES.md]: Fixed, thanks!

**Comment by llucax** [comment]
Fixed the comments, I will do a force-merge if I don't get an approval after the CI finished so I can release.

**Review by daniel-zullo-frequenz — APPROVED** [review]

**Pull request merged**